CefSharp/Internals/JavascriptObjectRepository.cs

```diff
@@ -182,6 +183,46 @@ private void AnalyseObjectForBinding(JavascriptObject obj, bool analyseMethods,
                 return;
             }
 
+            var dict = obj.Value as IDictionary;
+            if (dict != null)
+            {
+                // in case of dictionary, add each key/value pair as read only property of obj                              
+                foreach (var key in dict.Keys)
```

Comment:
Can you just perform a straight `foreach` on the dictionary and access it's `KeyValuePair` objects rather than getting the key then performing the lookup?